newswires/app/conf/SearchPresets.scala

```diff
@@ -157,14 +158,227 @@ object SearchPresets {
     )
   )
 
+  private val ReutersBusiness = List(
+    SearchParams(
+      text = None,
+      suppliersIncl = List("REUTERS"),
+      categoryCodesIncl = List(
```

Comment:
if codes in `categoryCodesIncl` list are joined with 'or', why do we have two separate sets of `SearchParams` for Reuters here?